core/templates/pages/topic-editor-page/subtopic-editor/subtopic-preview-tab.directive.html

```diff
@@ -0,0 +1,57 @@
+<h3>
+  Back to <a ng-click="navigateToSubtopic()"><[editableTitle]></a>
```

Comment:
Have this be a button, it is not clear if it is clickable or not. Also, align it with the left of the box in the window.
![image](https://user-images.githubusercontent.com/22347970/85408815-a5c8d600-b582-11ea-87fe-078bbc5b0fdd.png)